js/main.js

```diff
@@ -322,13 +322,21 @@ function removeMovie(event) {
   } emptyText();
 }
 
-window.addEventListener('load', spinner);
+var $offline = document.querySelector('.offline');
+window.addEventListener('offline', function (event) {
+  $offline.className = 'offline';
+});
 
-function spinner(event) {
-  var $loading = document.querySelector('.loading');
-  $loading.className = 'loading hidden';
+var $offlineButton = document.querySelector('.offline-button');
+
+$offlineButton.addEventListener('click', closeOfflineDiv);
+
+function closeOfflineDiv(event) {
+  $offline.className = 'offline hidden';
 }
 
-window.addEventListener('offline', function (event) {
-  alert('Sorry, there was an error connecting to the network! Please check your internet connection and try again.');
-});
+window.addEventListener('load', spinner);
+function spinner(event) {
+  var $hourglass = document.querySelector('.lds-hourglass');
+  $hourglass.className = 'hidden';
+}
```

Comment:
Why is this necessary? Just make it hidden in the HTML, like I mentioned in my last review